topic7/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>netgloo</groupId>
+  <artifactId>spring-boot-mysql-springdatajpa-hibernate</artifactId>
```

Comment:
This one right? https://github.com/netgloo/netgloo-blog/blob/master/spring-boot-mysql-springdatajpa-hibernate/pom.xml